tools/clang/test/CodeGenSPIRV/spirv.interface.empty-struct.vs.hlsl

```diff
@@ -0,0 +1,15 @@
+// Run: %dxc -T vs_6_0 -E main
+
+// There is no interface variable for VSIn or VSOut empty structs.
+
+// CHECK: OpEntryPoint Vertex %main "main" %gl_PerVertexOut
```

Comment:
As discussed, this won't error out even if we have more interface variables listed after `gl_PerVertexOut`. Should use whole file check. :)